thirdeye-ui/src/app/locale/languages/en-us.json

```diff
@@ -218,7 +218,9 @@
         "has-feedback": "Has Feedback",
         "health-status": "Health Status",
         "heatmap": "Heatmap",
+        "heatmap-and-dimension-drills": "Heatmap & Dimension Drills",
         "heatmap-data": "Heatmap Data",
+        "heatmap-of-dimension-distribution": "Heatmap of Dimension distribution",
```

Comment:
nit. `Heatmap of Dimension Distribution`? Not sure the strategy used in TE for capitalization vs non-capitalization.